lib/src/blocs/account_home_bloc.dart

```diff
@@ -0,0 +1,67 @@
+import 'dart:async';
+
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:reazzon/src/chat/base_bloc/base_bloc.dart';
+
+class AccountHomeBloc extends BLOCBase {
+  @override
+  initialState() => null;
+
+  @override
+  Stream mapEventToState(event) => null;
+
+  Stream<List<AccountHomeEntity>> users() {
+    final usersCollection = Firestore.instance.collection('Users');
+
+    return usersCollection.snapshots().map((snapshot) {
+      return snapshot.documents
+          .map((doc) => AccountHomeEntity.fromSnapshot(doc))
+          .toList();
+    });
+  }
+
+  Stream<List<AccountHomeEntity>> filterUsers(List<String> reazzons) {
+    final usersCollection = Firestore.instance.collection('Users');
```

Comment:
Please use the repository provided in the service folder, `user_repository.dart`. If there is no method that you could use, you can modify the repository by adding the method you need. Also please use async function when going to firebase, so that you will not block the UI thread

Reply:
Okay I added method to the user_repository.dart